randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -370,7 +342,7 @@ Extra - asset_id: collision_camera_007
   > Door to Cold Room (Small)
       Trivial
   > Tunnel to Separate Tunnels Room (Upper)
-      Morph Ball
+      Trivial
```

Comment:
Lay Power Bomb

Reply:
This one is handled on the other side, in Separate Tunnels